concordium-node/src/grpc2.rs

```diff
@@ -403,12 +397,16 @@ pub mod server {
     impl service::queries_server::Queries for RpcServerImpl {
         type GetAccountListStream =
             futures::channel::mpsc::Receiver<Result<Vec<u8>, tonic::Status>>;
+        type GetAncestorsStream = futures::channel::mpsc::Receiver<Result<Vec<u8>, tonic::Status>>;
```

Comment:
We might consider documenting these. Although, I don't know if it is important.

Reply:
I think it is easy to do and will provide value for non-expert users of tonic. Just a line saying something like
"Return value for the `get_ancestors` endpoint."